File: app/src/main/java/fergaral/datetophoto/fragments/AboutFragment.java
package fergaral.datetophoto.fragments;

import android.content.pm.PackageManager;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import fergaral.datetophoto.R;

/**
 * Created by Parejúa on 21/10/2014.
 */
public class AboutFragment extends Fragment {

    public AboutFragment() {
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_about, container, false);
        Toolbar toolbar = (Toolbar) rootView.findViewById(R.id.my_toolbar);
        AppCompatActivity appCompatActivity = (AppCompatActivity) getActivity();
        appCompatActivity.setSupportActionBar(toolbar);

        if(appCompatActivity.getSupportActionBar() != null)
            appCompatActivity.getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        TextView versionTv = (TextView) rootView.findViewById(R.id.tvabout2);
        String versionName = "";
        try {
            versionName = getActivity().getPackageManager().getPackageInfo(getActivity().getPackageName(), 0).versionName;
        } catch (PackageManager.NameNotFoundException e) {
            e.printStackTrace();
        }
        versionTv.setText(versionName);
        return rootView;
    }
}

File: app/src/main/java/fergaral/datetophoto/fragments/AboutFragment.kt
package fergaral.datetophoto.fragments

import android.content.pm.PackageManager
import android.os.Bundle
import androidx.fragment.app.Fragment
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

import fergaral.datetophoto.R

/**
 * Created by Parejúa on 21/10/2014.
 */
class AboutFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        val rootView = inflater.inflate(R.layout.fragment_about, container, false)
        val toolbar = rootView.findViewById<View>(R.id.my_toolbar) as Toolbar
        val appCompatActivity = activity as AppCompatActivity?
        appCompatActivity!!.setSupportActionBar(toolbar)

        if (appCompatActivity.supportActionBar != null)
            appCompatActivity.supportActionBar!!.setDisplayHomeAsUpEnabled(true)

        val versionTv = rootView.findViewById<View>(R.id.tvabout2) as TextView
        var versionName = ""
        try {
            versionName = activity!!.packageManager.getPackageInfo(activity!!.packageName, 0).versionName
        } catch (e: PackageManager.NameNotFoundException) {
            e.printStackTrace()
        }

        versionTv.text = versionName
        return rootView
    }
}
